Jenkinsfile

```diff
@@ -9,8 +8,6 @@ def postgres_dir = "postgres.${postgres_version}"
 def postgres_targzfile = "postgres.${postgres_version}.tar.gz"
 def postgres_license = "PostgreSQL_License.txt"
 def rename_license = "${postgres_dir}/license.txt"
-def release_info_template_file = "release_info_template.json"
-def release_info_file = "release_info.json"
```

Comment:
I think that removing this line means that we also should remove line [64](https://github.com/quipucords/quipucords/pull/1962/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3L64)

Because renamed the template to `release_info.json` so it may still work.

Reply:
Fixed